rand_chacha/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2018-09-??
+- Initial release
```

Comment:
Probably worth saying this is just pulled out of the Rand crate

Reply:
I'll do that for all the RNG crates.